style.css

```diff
@@ -0,0 +1,242 @@
+*, *::after, *::before {
+    box-sizing: border-box;
+} 
+
+body {
+    font-size: 10px;
+    font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
+    display: flex;
+    flex-direction: column;
+    background-color: #000000;
+}
+
+header {
+    background-image: url(images/northernLights.jpg);
+    background-size: cover;
+    background-position: center;
+}
+
+a {
+    text-decoration: none;
+}
+
+a:hover {
+    background-color: #1B8C20;
+}
+
+h1 {
+    font-size: 4rem;
+    text-align: center;
+    font-weight: 600;
+}
+
+.navLinks {
+    display: flex;
+    justify-content: center;
+}
+
+.navLinks a {
+    line-height: 2rem;
+    color: #000000;
+    padding: 2rem;
+    font-size: 2rem;
+}
+
+.jumbotron {
+    font-size: 1.5rem;
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    padding: 4rem;
+}
+
+.decorativeBox img {
+    max-height: 25rem;
+    border-radius: 200px;
+}
+
+.jumbotronTextbox {
+    max-width: 30rem;
+    margin-left: 4rem;
+    text-align: center;
+}
+
+h2 {
+    color: #ffffff;
+    background-color: #000000;
+}
+
+.jumbotron p {
+    display: inline;
+    font-size: 1.4rem;
+    color: #000000;
+    background-color: #1B8C20;
+    line-height: 2.8rem;
+}
+
+.jumbotronButton {
+    font-weight: bold;
+    margin: 4rem;
+    padding: 1rem;
+    border-radius: 25px;
+    border: 0.5rem solid #1B8C20;
+    color: #ffffff;
+    background-color: #000000;
+}
+
+.jumbotronButton a:hover {
+    color:#000000
+}
+
+.aboutMe, .portfolioBlurb, .contactSection {
+    background-color: #1B8C20;
+    font-size: 2rem;
+    margin-left: 6rem;
+    margin-right: 6rem;
+
+}
+
+.aboutMe h2, .portfolioBlurb h2, .contactSection h2 {
+    padding-left: 1.6rem;
+}
+
+.aboutMe p, .portfolioBlurb p, .contactSection p {
+    padding: 1.6rem;
+    line-height: 1.7;
+}
+
+.portfolioHighlightsContainer {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-evenly;
+    padding: 5rem 2rem 3.5rem 2rem;
+}
+
+.portfolioHighlight {
+    margin: 0rem 1rem 0rem 1rem;
+    text-align: center;
+}
+
+.portfolioHighlight a {
```

Comment:
ah, i see what your trying to do here. this wont work on an anchor though as they are inline elements you cannot set a width and height on them. instead they just go to the size of the content, so wrap them round the image and h3 and then remove this css and it should be good